src/dms/core/EdFi.DataManagementService.Core/ApiSchema/ApiSchemaSchemaProvider.cs

```diff
@@ -3,9 +3,9 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-using Microsoft.Extensions.Logging;
-using Json.Schema;
 using System.Reflection;
```

Comment:
My IDE is suggesting these remain added to provide context for JsonSchema and Assmebly symbols that are used here.

Reply:
Still there on lines 7 and 8 below.